content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -627,25 +627,25 @@ parameters:
 
 ```
 
-* `fs`: filesystem to be laid out: [none/xfs/ext4] (default: `ext4`).
-* `block_size`: block size in Kbytes (default: `32`).
+* `fs`: filesystem to be laid out: `none/xfs/ext4` (default: <em>ext4</em>).
+* `block_size`: block size in Kbytes (default: <em>32</em>).
 * `repl`: number of synchronous replicas to be provided in the form of
-  replication factor [1..3] (default: `1`) A string is expected here i.e.
+  replication factor `1..3` (default: <em>1</em>) A string is expected here i.e.
   `"1"` and not `1`.
 * `io_priority`: determines whether the volume will be created from higher
-  performance or a lower priority storage [high/medium/low] (default: `low`).
```

Comment:
The `</strong` doesn't look correct

Reply:
Thank you so much. Just removed </strong

Reply:
@msau42 Please take a look.